databags/navigation.ini

```diff
@@ -0,0 +1,51 @@
+;; Hier gibt es die Übersetzungen für die Toolbox Webseiten Navigation
+
+[de]
+home = Start
```

Comment:
Warum sind in anderen .ini Files die translations in Quotes und in manchen nicht

Reply:
Da beides technisch richtig ist. ist nun vereinheitlicht.